.github/workflows/ci.yml

```diff
@@ -44,7 +54,26 @@ jobs:
           docker push $REGISTRY/$REPOSITORY:latest
 
       - name: Start the synchronization tool
-        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
         run: |
           aws --region us-west-1 lambda invoke --function-name start-sync-team output.json
           cat output.json | python3 -m json.tool
+
+  # Summary job for the merge queue.
+  # ALL THE PREVIOUS JOBS NEED TO BE ADDED TO THE `needs` SECTION OF THIS JOB!
+  conclusion:
```

Comment:
should we add this somewhere in the github settings? should we do it in the team repo?

Reply:
The team repo already configures `CI` as a required job (that's why I named the conclusion job `CI`). So hopefully it should work how it is currently.